docs/source/api/communities/xgi.communities.spectral.rst

```diff
@@ -0,0 +1,11 @@
+xgi.communities.spectral
+========================
+
+.. currentmodule:: xgi.communities.spectral
+
+.. automodule:: xgi.communities.spectral
+
+   .. rubric:: Functions
+
+   .. autofunction:: spectral_clustering
+   .. autofunction:: _kmeans
```

Comment:
I think by convention in Python functions starting with `_` are "private" and not meant to be documented. And since it's not in the `__all__` it might not even work?